corehq/apps/data_dictionary/util.py

```diff
@@ -392,6 +392,10 @@ def is_case_type_deprecated(domain, case_type):
         return False
 
 
+def is_case_type_unused(domain, case_type):
+    return CaseSearchES().domain(domain).case_type(case_type).count() == 0
```

Comment:
I assume the point of this PR is that this query is basically trivial, right? As opposed to the `get_used_props_by_case_type` which is quite expensive?